copy.go

```diff
@@ -71,16 +72,16 @@ func structToStruct(filter FieldFilter, src, dst *reflect.Value, userOptions *op
 
 		for i := 0; i < src.NumField(); i++ {
 			srcType := src.Type()
-			fieldName := srcType.Field(i).Name
+			srcName := dstKey(userOptions.SrcTag, srcType.Field(i))
```

Comment:
Please, rename the `dstKey` function to `fieldName` as it is no longer dst specific